src/main/java/com/powsybl/ws/commons/computation/config/JacksonConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.powsybl.ws.commons.computation.config;
+
+import org.springframework.boot.autoconfigure.jackson.Jackson2ObjectMapperBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import com.powsybl.commons.report.ReportNodeJsonModule;
+
+@Configuration
+public class JacksonConfig {
+    @Bean
+    public Jackson2ObjectMapperBuilderCustomizer jsonCustomizer() {
+        return builder -> builder.build().registerModule(new ReportNodeJsonModule());
```

Comment:
In each calculation service, ReportNodeJsonModule has been added in the bean RestTemplate, eg. https://github.com/gridsuite/security-analysis-server/blob/main/src/main/java/org/gridsuite/securityanalysis/server/RestTemplateConfig.java#L53

In my opinion, if we do not move ReportServiceTest into this library, we can remove this config. Otherwise, if we want to move the test class, we can configure ReportService by constructor (not by autowire) which passes object mapper and restTemplate which are configured with ReportNodeJsonModule

Reply:
It just miss to be referenced in `PowsyblWsCommonAutoConfiguration` and `PowsyblWsCommonProperties`.

Reply:
@Tristan-WorkGH I don't understand your solution, while the object mapper for rest template is created here in each micro-service, for example: https://github.com/gridsuite/security-analysis-server/blob/main/src/main/java/org/gridsuite/securityanalysis/server/RestTemplateConfig.java#L48

Reply:
I indeed looks useless but I am a bit tired of retesting all the 6 webservices. Are you sure that I can remove it ?